### DISCUSSION TITLE
Github Contribution merge

### DISCUSSION THREAD

**Discussion by roni6ch** [community/community#42303] [New to GitHub]
Hey is there a way to save my private work account GitHub contributions into my personal GitHub account?
If the only option to do it is by deleting the work account then ill do it.

Thanks.
![image](https://user-images.githubusercontent.com/9461540/208934862-74dd7b4b-b42d-4ddf-ae4e-0b4dec216e4b.png)

**Comment by airtower-luna** [comment]
Commits are assigned to an account based on the author email address, so you can just add the right email address to your personal account. Of course you'll first have to remove it from the work account, and if it's the only email on that you'll have to either delete the account or add another address first.
